conseil-common/src/main/scala/tech/cryptonomic/conseil/common/config/Platforms.scala

```diff
@@ -38,7 +38,22 @@ object Platforms extends LazyLogging {
     * Should associates each platform to a list of its internally defined configuration type, matching the
     * inner `platform.ConfigurationType`
     */
-  case class PlatformsConfiguration(platforms: Map[BlockchainPlatform, List[PlatformConfiguration]])
+  case class PlatformsConfiguration(platforms: Map[BlockchainPlatform, List[PlatformConfiguration]]) {
+
+    /*** Extracts platforms from configuration */
+    def getPlatforms: List[Platform] =
```

Comment:
This could be a `Set`

Reply:
but ignore this, if you're only moving a piece of code around

Reply:
Yea, I'm making some changes if they are needed. This change does not apply to much changes, so I can do that :)

Reply:
I was trying to do that and funny enough this change makes the endpoint broken, mostly because rest of the methods that uses it, expects List to be passed. While changing to Set I would need to change it everywhere. Just thinking that this change might be to big and I don't know how to solve missing implicit in endpoints for `Response[Set[Entity]]`. I could cut this off at some point and convert back to list at the "closest" possible level. What do you think?

Reply:
Refrain from doing too much refactor work, at this point is not that relevant.

Thanks for trying, don't worry about that